src/js/components/Select/doc.js

```diff
@@ -214,10 +215,12 @@ export const doc = Select => {
           PropTypes.number,
         ]),
       ),
-    ]).description(`Currently selected value. This can be an array
-      when multiple. Passing an element allows the caller to control how
-      the value is rendered. Passing an element is deprecated. Instead,
-      use the 'valueLabel' property.`),
+    ]).description(`Currently selected value. This should be an option from 
```

Comment:
It actually should be one of the `options` objects if `valueKey` has `reduce: true`. Perhaps here we can just refer to needing to match whatever the `valueKey` results in for the selected option. We might need to improve the description of `valueKey` to be clearer. I think it's so-so at the moment.